cypress/integration/smoke/BasicTest.ts

```diff
@@ -58,8 +55,7 @@ describe("Basic Elements", () => {
 
   it("Instance header details are available", () => {
     const instanceHeaderDetails = ["Name", "Time created", "Status"];
-    cy.ouiaId("Instances list table", "PF4/Table")
-      .ouiaType("PF4/TableRow")
+    cy.ouiaType("PF4/Table")
```

Comment:
We can rewrite this command to use (data-testid and ouiaType)

Reply:
Looking at `ResponsiveTable`, it seems that it is not currently supporting `data-testid` for the `table` tag either.